src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_section_jump.scss

```diff
@@ -0,0 +1,132 @@
+            </section>
+            <section id="section-jump-chapter-3" class="section-jump-chapter">
+                <h2 class="section-jump-chapter-header">Heading 3</h2>
+                Lorem ipsum Ullamco laboris commodo sit commodo nulla adipisicing in fugiat nostrud mollit irure esse velit reprehenderit commodo dolor.
+                Lorem ipsum Voluptate eu sunt sed proident in incididunt cupidatat pariatur voluptate elit cupidatat pariatur aliqua Ut nostrud ut ullamco dolor exercitation cupidatat ea anim consectetur irure ea ad pariatur nulla sunt qui aliqua deserunt in laboris exercitation labore reprehenderit amet cillum eu Excepteur eiusmod dolor exercitation sed deserunt cupidatat cillum ad laboris et incididunt reprehenderit ad sed voluptate reprehenderit commodo aute consequat fugiat ea pariatur laborum laborum qui labore et et do magna sint occaecat ullamco aute nulla nulla dolore commodo eiusmod reprehenderit velit incididunt non anim irure nisi nulla pariatur est consequat dolor esse ex in ad in velit sed commodo Ut ut irure ut dolore magna adipisicing deserunt do fugiat dolore quis laboris eu velit sint officia mollit dolor sed mollit minim sed voluptate est elit incididunt anim in aliqua id sit minim in dolore in ullamco commodo velit id sed ut amet Ut exercitation adipisicing in nisi reprehenderit et fugiat esse in dolore ut eiusmod reprehenderit elit ea enim consectetur sit nostrud enim nisi irure labore magna amet in mollit Excepteur ex ea nisi aliquip anim enim laboris proident dolor culpa dolor ut consequat ea Excepteur ut ex quis proident occaecat id ex dolor in aliqua officia velit quis in Duis tempor aliqua consectetur magna sed nostrud velit reprehenderit laborum fugiat qui dolor enim fugiat aute elit officia nostrud irure eu ut pariatur velit in non elit sint reprehenderit dolore enim consequat consequat enim ad nisi minim.
+            </section>
+        </div>
+    </div>
+</div>
+```
+*/
+
+.section-jump-cover-header {
+    @include text(header);
+    margin: 0;
+
+    @media print {
+        margin-bottom: 1em;
+    }
+}
+
+.section-jump-cover {
+    @include span(full);
+    border-bottom: 2px solid $color-structure-normal;
+    padding: 1.07142857em 0.71428571em 0.71428571em;
+
+    @media print {
+        border: 0;
+        padding: 0;
+    }
+}
+
+.section-jump-body {
+    background-color: $color-background-base-introvert;
+    clear: left;
+}
+
+.section-jump-navigation {
+    @extend .jump-navigation;
+    padding-top: 1em;
+    text-align: center;
+    width: $section-jump-navigation-width;
+    float: left;
+
+    .section-jump-navigation-list-container {
+        padding: 0;
+    }
+
+    .section-jump-navigation-list-item {
+        a {
+            margin: 0 0 1em;
+        }
+    }
+
+    @media print {
+        display: none;
+    }
+}
+
+.section-jump-chapters {
```

Comment:
I don't like this name. IMHO it is either a section or a chapter. Using both terms is confusing.